src/components/artwork/ArtworkSnapshot.tsx

```diff
@@ -222,16 +244,34 @@ const ArtworkSnapshot = ({
                 >
                   <Description>
                     {isFirstElement
-                      ? "Your personal stamp"
-                      : `Snapshot when ${
+                      ? "ETHDenver stamp collection NFT"
+                      : `Collection when ${
                           person ? `you met ${name}` : `you went to ${name}`
                         }`}
                   </Description>
                   <Label className="text-center">
                     {isFirstElement && signatures.length > 1
-                      ? "Navigate to see your stamp collection develop!"
+                      ? "Starts with your personal stamp"
```

Comment:
```suggestion
                      ? "Your personal stamp"
```

@RiverRuby   maybe just `Your personal stamp` here